test/cli/cli_test.hpp

```diff
@@ -6,6 +6,8 @@
 #include <sstream>               // ostringstream
 #include <string>                // strings
 
+#include <seqan3/test/expect_range_eq.hpp> // EXPECT_RANGE_EQ macro
```

Comment:
Should we mention why this is a useful macro, so that other people are more likely to use it?

Reply:
Yes, good point!